firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+WebViewDelegates.swift

```diff
@@ -778,6 +787,31 @@ extension BrowserViewController: WKNavigationDelegate {
         decisionHandler(.allow)
     }
 
+    func handlePDFResponse(_ webView: WKWebView,
```

Comment:
Could we pass in a `TabWebView` directly to work with in this function?
```suggestion
    func handlePDFResponse(_ webView: TabWebView,
```